azure-iot-device/tests/common/pipeline/test_pipeline_stages_base.py

```diff
@@ -23,8 +23,6 @@
 )
 from tests.common.pipeline import pipeline_stage_test
 
-logging.basicConfig(level=logging.INFO)
```

Comment:
I don't think I should have removed this, actually. These logs only appear on failure. Which actually suggests that __EVERY___ test file should probably have this. Separate PR?

Reply:
Yes!  Pipeline debugging relies heavily on these logs and I think we should do a much better job of adding logging to our code for post-mortem debugging.

---
In reply to: [307434154](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307434154) [](ancestors = 307434154)

Reply:
Okay let's make a separate PR for this - I'll put it on the backlog

---
In reply to: [307827509](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307827509) [](ancestors = 307827509,307434154)